apps/command-palette-wp-admin/webpack.config.js

```diff
@@ -36,8 +36,11 @@ function getWebpackConfig( env, argv ) {
 				requestToExternal( request ) {
 					// The extraction logic will only extract a dependency if requestToExternal
 					// explicitly returns undefined for the given request. Null shortcuts the
-					// logic such that @wordpress/commands styles are bundled.
-					if ( request === '@wordpress/commands/build-style/style.css' ) {
+					// logic such that @wordpress/commands and @wordpress/react-i18n styles are bundled.
```

Comment:
Technically, we are bundling the `@wordpress/commands` styles and the full `@wordpress/react-i18n` package.
```suggestion
					// logic such that @wordpress/commands styles and @wordpress/react-i18n are bundled.
```

Reply:
Oops, I fixed it. Thanks!